recipe/meta.yaml

```
@@ -74,10 +77,12 @@ outputs:
         - automake  # [not win]
       host:
         - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
+        # we need headers from cuda-nvcc-dev_win-64, e.g. crt/host_defines.h
+        - {{ compiler('cuda') }}                    # [(cuda_compiler_version or "").startswith("12") and win]
         - cuda-profiler-api                         # [(cuda_compiler_version or "").startswith("12")]
         - cuda-nvtx-dev                             # [(cuda_compiler_version or "").startswith("12")]
```

If this is just using NVTX 3 (as appears may be the case), would suggest using `nvtx-c`. This is a header only library that only includes NVTX 3 support (so no NVTX 2 as `cuda-nvtx-dev` includes)

```suggestion
        - nvtx-c                                    # [(cuda_compiler_version or "").startswith("12")]
```

---

Thanks for the tip!